R/estimate_infections.R

```diff
@@ -82,7 +82,7 @@
 #'                            stan_args = list(warmup = 200,
 #'                             control = list(adapt_delta = 0.95, max_treedepth = 15),
 #'                             cores = ifelse(interactive(), 4, 1)),
-#'                             verbose = interactive())
+#'                            verbose = interactive())
```

Comment:
The old formatting was ok.

Reply:
It was actually out by a space (due to the stan_args secondary list). I went through and checked this (very fun job) so what is on master should now be correctly formatted.